data/ScaleFactors/Electron_EGamma_SF2D_gsfTracking.json

```diff
@@ -0,0 +1,242 @@
+{
```

Comment:
The original file is in the same format as others but seems to have only one bin in pt... it's probably the root cause of all this mess